a10_neutron_lbaas/v2/handler_listener.py

```diff
@@ -67,6 +95,7 @@ def _set(self, set_method, c, context, listener):
 
         if 'server_ssl' in templates:
             server_args = {'server_ssl_template': templates['server_ssl']}
+            template_args["template-server-ssl"] = template_name
```

Comment:
`template_args["template-client-ssl"]` is never set and `'server_ssl' in templates` is always false so neither `template_args["template-client-ssl"]` nor `template_args["template-server-ssl"]` is ever set.